src/collective/cover/testing.py

```diff
@@ -116,7 +116,7 @@ def generate_jpeg(width, height):
 
 # FIXME: workaround for https://github.com/plone/plone.app.testing/issues/39
 #        Products.TinyMCE is used only in Plone 4
-if not IS_PLONE_5:
+if not DEXTERITY_ONLY:
```

Comment:
The workaround must be present in all test of Plone 4. Not only in those that don't have dexterity.

But I still think than plone/plone.app.testing#39 should be fixed. Please do a PR there to see what happens.

Reply:
Ops. If [this](https://travis-ci.org/collective/collective.cover/jobs/414256229) test passed then the workaround is only necessary for dexterity.